Add `disableCssProcessing` option for webpack

evoactivity opened the pull request:
When an app provides it's own webpack css pipeline there is no reason for the default webpack config to also process css files, you can end up with multiple versions of the same file output into the build (eg. renaming the css files by configuring MiniCssExtractPlugin yourself).

I'm not sure how to add a test for this, some guidance here would be helpful. Do I add a new scenario? Do I just add a test to an existing scenario?

Some documentation should be added to explain this option.

mansona commented:
I just noticed that you're targeting main here 🙈 main Webpack support doesn't work yet and it won't work this way any more, it will have its own webpack config when we eventually release it

evoactivity commented:
Yeah, I switched to stable and there were 388 files in this PR 🙈. So I changed back until I had time to merge stable back in locally and push up a correct PR.

evoactivity commented:
@mansona I've got this targeting the stable branch now.
What would be the best way to test this? I don't see any current tests for things like `styleLoaderOptions`, or `cssLoaderOptions`.

NullVoxPopuli commented:
Is this still needed, given we have https://github.com/embroider-build/embroider/pull/2299 now?

evoactivity commented:
I think so, you still end up with two rules processing css files when you use a custom pipeline, doubling the workload.

evoactivity commented:
I've added documentation to the readme, the failing tests I'm pretty sure are just flaky windows issues.

evoactivity commented:
@mansona Anything else for me to do on this?
